### PR TITLE
fixes #16480 triangular_solve PR

### LINKED ISSUE #16480: triangular_solve

### PR DESCRIPTION
**Pull request by eekamal**
Closes #16480

### CODE REVIEW

**Comment by eekamal** [comment]
this commit fixes #16480
issue: < triangular_solve> #16480

**Review by rishabgit — CHANGES_REQUESTED** [review]
Hi! Thanks a lot for working on this 😄. Could you please add a comment on the original Task board (https://github.com/unifyai/ivy/issues/1546) so that your PR gets correctly linked? :)
Feel free to refer to the docs on the comment format - https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues

The current way of hard-coding values is not in line with ivy's testing pipeline. Could you please look into that as well? Check out how the other tests are done in that same file and also, here's a link to the docs on frontend testing to help you out - https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html

**Review by rishabgit — CHANGES_REQUESTED** [review]
Hey, you forgot to add the proper comment as I asked in the previous review and your work wasn't linked correctly. This resulted in the issue staying open and someone else had picked it up - https://github.com/unifyai/ivy/issues/16579
I've linked your work manually and notified the other person of this situation and asked them to pick another issue since you started working on this already but in the future, please make sure you follow the guidelines [here](https://lets-unify.ai/ivy/contributing/1_the_basics.html#todo-list-issues)

We shouldn't create explicit backend implementation here (even if we do that, triangular_solve isn't equivalent to solve fn that you're using in ivy.triangular_solve).  I'd recommend looking into the other frontend implementations and the docs to deep dive on this - https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html, https://unify.ai/docs/ivy/overview/deep_dive/function_types.html.

File: ivy/functional/frontends/tensorflow/linalg.py
Comment (rishabgit) [inline ivy/functional/frontends/tensorflow/linalg.py:1]: ivy.solve_triangular doesn't exist. please check out the deep dive, specifically the doc links sent in this pr discussion

**Review by rishabgit — CHANGES_REQUESTED** [review]

**Pull request closed without merging**